src/components/Steps/Step.js

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import cx from 'classnames'
+
```

Comment:
We don't separate imports by new line, please remove all empty new lines in import section.